docs/decisions/0003-consolidation-into-frontend-platform.rst

```diff
@@ -0,0 +1,71 @@
+Consolidation of libraries into frontend-platform
+=================================================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+The frontend-platform repository replaces five earlier repositories:
+
+- edx/frontend-analytics
+- edx/frontend-auth
+- edx/frontend-base
+- edx/frontend-i18n
+- edx/frontend-logging
+
+We found that development across these five libraries was growing very difficult in a number of ways.
+
+Dependency graph complexity
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The overhead involved in maintaining individual packages and semantic version numbers was very difficult to reason and maintain.  Updates to low-level packages tended to cause cascading updates through the dependency tree.  If a breaking change or incompatibility present in the tree forces dependent packages to absorb unrelated work, it can be very difficult to realize it's happening,  decreasing developer velocity and destabilizing our applications.
+
+As a rough illustration of this complexity:
+
+- 12 Micro-frontends depend on 5 libraries. (60 edges max)
+- 12 Micro-frontends depend on 5 reusable organisms (headers, footers, Paragon) (60 edges max)
```

Comment:
On the twelfth day of coding my teammate gave to me
Twelve micro-frontends
Eleven closed tickets
Ten builds a-passing 
Nine all hands meetings
Eight config changes
Seven apps a-shipping
Six coders coding
Five free lunches
Four new translations
Three ADRs
Two on-call tickets
And a frontend with a libraryyyyyy!